lib/chef/application/client.rb

```diff
@@ -238,26 +238,26 @@ class Chef::Application::Client < Chef::Application
   option :enable_reporting,
     short: "-R",
     long: "--enable-reporting",
-    description: "Enable reporting data collection for #{Chef::Dist::PRODUCT} runs",
+    description: "Enable reporting data collection for #{Chef::Dist::PRODUCT} runs.",
     boolean: true
 
   option :local_mode,
     short: "-z",
     long: "--local-mode",
-    description: "Point #{Chef::Dist::CLIENT} at local repository",
+    description: "Point #{Chef::Dist::CLIENT} at local repository.",
     boolean: true
 
   option :chef_zero_host,
     long: "--chef-zero-host HOST",
-    description: "Host to start chef-zero on"
+    description: "Host to start chef-zero on."
 
   option :chef_zero_port,
     long: "--chef-zero-port PORT",
-    description: "Port (or port range) to start chef-zero on.  Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works."
+    description: "Port (or port range) to start chef-zero on. Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works."
```

Comment:
```suggestion
    description: "Port (or port range) to start chef-zero on. Port ranges like 1000, 1010 or 8889-9999 will try all given ports until one works."
```

Reply:
I think we need the original formatting actually